changelog.md

```diff
@@ -10,6 +10,9 @@ _Nothing yet!_
   - Added new outputs for the iOS and Android platforms in `bpk-tokens/tokens/ios/base.ios.json`, `bpk-tokens/tokens/ios/base.react.native.es6.js`, `bpk-tokens/tokens/android/base.android.xml` and
   `bpk-tokens/tokens/android/base.react.native.es6.js`. These interfaces are not to be considered stable yet however and they may still change.
 
+## 2017-08-16 - New icon for cloakroom
```

Comment:
This should not be dated as we have not released the new icon.

Your changelog entry should simply be this under the `## Unreleased` title

```

**Added:**
- bpk-component-icon:
- bpk-mixins:
- bpk-svgs:
  - added 1 new icon (cloakroom)
```